bridge/setu/processor/staking.go

```diff
@@ -73,7 +73,7 @@ func (sp *StakingProcessor) sendValidatorJoinToHeimdall(eventName string, logByt
 		if len(signerPubKey) == 64 {
 			signerPubKey = util.AppendPrefix(signerPubKey)
 		}
-		if isOld, _ := sp.isOldTx(sp.cliCtx, vLog.TxHash.String(), uint64(vLog.Index)); isOld {
+		if isOld, _ := sp.isOldTx(sp.cliCtx, vLog.TxHash.String(), uint64(vLog.Index), "staking"); isOld {
```

Comment:
I can see `staking` hardcoded at multiple places in this file
Let's create a global variable in this file with `eventType` and use that so that it's easy to update if required, And use the same in switch case to compare. Will look better
Do this for all event types.

Reply:
Sure. This sounds better. Will do the required changes.